### PR TITLE
Dependency resolution commons-lang3

### PR DESCRIPTION
**Pull request by vamsi-amazon**
### Description
Fixes build issue with commons-lang3
Force Resolution to 3.13.0 version.
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by YANG-DB — APPROVED** [review]

**Review by rupal-bq — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -113,6 +113,7 @@ allprojects {
     configurations.all {
         resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib:1.6.0"
         resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib-common:1.6.0"
+        resolutionStrategy.force 'org.apache.commons:commons-lang3:3.13.0'
```
Comment (reta) [inline build.gradle:116]: @vamsi-amazon I believe you need to exclude the `commons-lang3` since it is part of core :(

Comment (vamsi-amazon) [reply build.gradle:116]: @reta thanks will look into it.

Comment (vamsi-amazon) [reply build.gradle:116]: @reta  can i find all the dependencies in core at one place? 
should i go through all the gradle files or build and look into libs folder?

Comment (reta) [inline build.gradle:116]: Could you try please that:

```suggestion
        exclude(group: "org.apache.commons", module: "commons-lang3")
```

**Comment by reta** [comment]
@vamsi-amazon please hold on with this change, the revert is coming https://github.com/opensearch-project/OpenSearch/pull/9833/

**Comment by vamsi-amazon** [comment]
Closing as this is not required anymore.

**Pull request closed without merging**